src/main/scala/scorex/transaction/Transaction.scala

```diff
@@ -31,6 +34,12 @@ object Transaction {
         Portfolio(balance = 0, lease = LeaseBalance.empty, assets = Map(asset -> fee))
       case (None, fee) => Portfolio(balance = fee, lease = LeaseBalance.empty, assets = Map.empty)
     }
+
+    def processingScript(state: SnapshotStateReader): Option[Script] = tx match {
```

Comment:
we can remove this func. used only for script cost, `state.accountScript(tx.sender)` is enough!